cea/interfaces/dashboard/api/project.py

```diff
@@ -102,94 +104,104 @@ class Scenarios(Resource):
     def post(self):
         """Create new scenario"""
         config = current_app.cea_config
+        temp_config = cea.config.Configuration()
```

Comment:
why does this need to be a temporary configuration object? you need to add a comment here to explain.

Reply:
After reading through it again, I don't think there is a need to do that here.